cdap-distributions/bin/build_artifact_upload.sh

```diff
@@ -71,11 +76,17 @@ function sync_build_artifacts_to_server () {
   _source=$1
   echo "source directories: ${_source}"
 
+  clean
+
   decho "identify packages"
   PACKAGES=$(find ${_source} -type f \( -name '*.rpm' -o -name '*.deb' \) | sort -u)
   decho "${PACKAGES}"
   decho ""
 
+  # prepare for bundling
+  mkdir -p ${PKG_PROCESSING_DIR}
+  for p in ${PACKAGES}; do cp ${p} ${PKG_PROCESSING_DIR}; done
```

Comment:
Why do you need this loop? You're copying multiple files to the same destination directory.